LICENSE

```diff
@@ -0,0 +1,22 @@
+(The MIT License)
+
+Copyright (c) 2015 Apiary LTD. <support@apiary.io>.
```

Comment:
So I am not quite sure which one is correct (inc vs ltd) will check with Ondrej or @Almad